src/control/cmd/dmg/utils.go

```diff
@@ -70,18 +71,18 @@ func sprintConns(results client.ResultMap) (out string) {
 func getConsent() bool {
 	var response string
 
-	fmt.Println("Are you sure you want to continue? (yes/no)")
+	log.Info("Are you sure you want to continue? (yes/no)\n")
```

Comment:
These don't actually belong in the log, do they? They're just user prompts

Reply:
That's a good question. One of the motivations behind this logging overhaul was to allow all of our tool output to be easily machine-parseable (e.g. with JSON output), but this is kind of a weird case. This might be a candidate for removal. If you don't supply the --force flag then it doesn't do anything destructive.

My concern with using fmt.Println() is that if we keep the prompt but our tools are wrapped and the caller is expecting JSON output, then this line will be confusing.

Reply:
I can see that. I'm fine with leaving it as-is - though it's a little weird from a logging perspective. Hopefully no one decides to script against our prompts, though.